slices/libmaxminddb0.yaml

```diff
@@ -0,0 +1,15 @@
+package: libmaxminddb0
+
+essential:
+  - libmaxminddb0_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents: 
+      /usr/lib/*-linux-*/libmaxminddb.so.*:
```

Comment:
Can we change this to `/usr/lib/*-linux-*/libmaxminddb.so.0*` instead of `/usr/lib/*-linux-*/libmaxminddb.so.*`?